lib/omnibus/packagers/deb.rb

```diff
@@ -459,5 +459,23 @@ def safe_architecture
         Ohai['kernel']['machine']
       end
     end
+
+    #
+    # Install the specified packages
+    #
+    # @return [void]
+    #
+    def install(packages, _ = NULL)
+      `apt-get update && apt-get install -y --force-yes #{packages}`
```

Comment:
Also, like what you do for RPMs, we could have a selective `apt-get update` that would grab updates from one source only: see https://github.com/DataDog/dd-agent/blob/master/packaging/datadog-agent/source/install_agent.sh#L145 for an example